### PR TITLE
scrub mozilla-it refs, setup GHA for CI/CD of Common Voice helm charts

### PR DESCRIPTION
**Pull request by cmharlow**
Jira ticket: https://mozilla-hub.atlassian.net/browse/SE-2206

What this PR does:
* Migrates common voice & sentence collector helm charts (with git history) from mozilla-it/helm-charts GitHub repository to here
* updates CI & CD to use GitHub actions & run https://github.com/helm/chart-testing for linting & e2e tests on charts, GitHub Pages - Helm publication steps for publishing helm charts via GitHub Actions

Manually testing the GitHub Pages / Helm Chart publication now.

### CODE REVIEW

**Review by jencohoon — APPROVED** [review]

**Comment by jencohoon** [comment]
@cmharlow - not sure if we need to / should make jenny a blocking reviewer on these mostly infra PRs?  Jenny is 1/2 time on CV at NVDIA
If we're going to need these tomorrow, the it should probably just be me jzinner

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "voice-automation"
```
Comment (phirework) [inline .github/workflows/release.yml:19]: Who has access to this account / how is it managed?

**Review by phirework — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (cmharlow) [reply .github/workflows/release.yml:19]: @phirework see https://mozilla-hub.atlassian.net/browse/OI-2097

Comment (cmharlow) [reply .github/workflows/release.yml:19]: fyi @jzinner too ☝️

**Pull request merged**